File: C-Abi-Bridge-Aot/Tests/CAbiBridgeAot.Tests/TestCrypto/TestCryptoPqc/CryptoPqcSignAlgo.cs



namespace michele.natale.Tests;


internal enum PqcSignAlgo : byte
{
  None = 0,

  ML_DSA,       //Stateless
  SLH_DSA,      //Stateless

  LMS,          //Stateful
  XMSS,         //Stateful
}

File: C-Abi-Bridge-Aot/Tests/CAbiBridge.Aot.Vb.Tests/TestCrypto/TestCryptoPqc/CryptoPqcSignAlgo.vb
Option Strict On
Option Explicit On


Namespace michele.natale.Tests
  Friend Enum CryptoPqcSignAlgo As Byte
    None = 0

    ML_DSA       'Stateless
    SLH_DSA      'Stateless

    LMS          'Stateful
    XMSS         'Stateful
  End Enum
End Namespace
